src/components/edttt_bsim.py

```diff
@@ -209,6 +209,7 @@ def wait(self, delay_in_ms):
         # pack a message : delay 4 bytes
         end_of_wait = delay_in_ms*1000 + self.last_t;
         self.ll_send(struct.pack('<BQ', self.COM_WAIT, end_of_wait));
+        self.read(1);
```

Comment:
could you add a comment here that we read this dummy byte, as this is the way for the bridge to tell us the wait time has been reached?